build.gradle

```diff
@@ -89,12 +101,29 @@ dependencies {
     compile group: 'io.jsonwebtoken', name: 'jjwt', version: '0.7.0'
     compile group: 'com.squareup.okio', name: 'okio', version: '1.11.0'
     compile group: 'org.mockito', name: 'mockito-all', version: '1.9.5'
+
+    //bitcoinj dependencies
+    compile group: 'com.google.protobuf', name: 'protobuf-java', version:'2.6.1'
+    compile group: 'com.google.code.findbugs', name: 'jsr305', version:'2.0.1'
+    compile group: 'com.google.guava', name: 'guava', version: '23.5-jre'
+
     testCompile group: 'junit', name: 'junit', version: '4.12'
+    compile "org.jetbrains.kotlin:kotlin-stdlib-jre8:$kotlin_version"
 }
 
 jacocoTestReport {
     reports {
         xml.enabled = true // coveralls plugin depends on xml format report
         html.enabled = true
     }
+}
+compileKotlin {
+    kotlinOptions {
+        jvmTarget = "1.8"
```

Comment:
This should be 1.6 to match Android, otherwise it'll generate non-compatible bytecode on API <21

Reply:
Thanks. Updated it.